queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND  escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name NOT IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT name FROM animals WHERE neutered = TRUE;
+SELECT name FROM animals WHERE name NOT IN ('Gabumon');
```

Comment:
- [ ] [OPTIONAL] I like your approach `NOT IN` I also wanted to inform you that there is something in Postgres equivalent to not equal `<>` so we could say `SELECT name FROM animals WHERE name <> 'Gabumon';`

Reply:
@Amrhub thank you for this tip.

Reply:
@david-lafontant You are welcome, I just loved you are using Postgres to it's maximum so wanted to add this to your robust knowledge 😄